apps/backend/test/project/writes.spec.ts

```diff
@@ -171,4 +163,107 @@ describe('ProjectCoreController (reads)', () => {
     expect(response.status).toBe(403);
     expect(response.body.message).toBe('User is not a member of this cluster');
   });
+
+  it('deletes project and all related data', async () => {
```

Comment:
Please wrap in a describe block with EP name. You can also wrap other stuff in this file because it probably follows old convetion